Sources/System/MachPort.swift

```diff
@@ -0,0 +1,327 @@
+/*
+ This source file is part of the Swift System open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift System project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+*/
+
+#if $MoveOnly && (os(macOS) || os(iOS) || os(watchOS) || os(tvOS))
+
+import Darwin.Mach
+
+public protocol MachPortRight {}
```

Comment:
If possible, that is if we know what comment to add, I'd prefer to land them with the PR so that we don't accidentally miss something when shipping. Alternatively, we could require that _everything_ has such a comment and the expand availability script will error for decls that don't have them, but I don't know if that's harder to do.

Reply:
That would indeed be nice, but it'd be tricky to implement it in a barebones shell script. 😛
FWIW, recent compiler builds seem to produce warnings when encountering a declaration without an `@available` attribute while building ABI stable code, so we may get this for free!